src/optimizer/plan_nodes/logical_copy_from_file.rs

```diff
@@ -18,13 +18,21 @@ pub struct LogicalCopyFromFile {
     format: FileFormat,
     /// The column types.
     column_types: Vec<DataType>,
```

Comment:
Now this field is redundant.
```suggestion
```

Reply:
It is used by `column_types()`, which returns the reference to `column_types` to caller. So, I think redundantly maintaining a `column_types` field is acceptable.